vuepress/docs/next/tutorials/devops/installation/azure-kubernetes-service/azure-install.md

```diff
@@ -123,7 +125,7 @@ curl -sfL https://github.com/entando-k8s/entando-helm-quickstart/archive/v6.2.0.
 helm template my-aks-app --namespace=entando ./ > my-aks-app.yaml
 ```
 5. Deploy Entando via `kubectl create -f my-aks-app.yaml`
-6. Watch Entando startup `kubectl get pods -n entando --watch`
+6. Watch Entando startup `watch kubectl get pods -n entando`
```

Comment:
@joewhite101 What was the thinking on this change? We have a couple dozen cases of kubectl --watch in the docs and in my testing watch isn't as flexible since it doesn't respect aliases.

Reply:
I will roll it back. I find the output of that structure a lot better but if it breaks things we can keep it as is.